packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -265,9 +265,9 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
 
       if (attachment.attachmentType === 'inlineImage' && attachment.url) {
         // TBD: Need to begin investigating how to download HQ images.
-        const blob = await adapter.downloadAttachments({ attachmentUrls: { [attachment.id]: attachment.url } });
-        if (blob[0]) {
-          const blobUrl = blob[0].blobUrl;
+        const blob = await adapter.downloadAttachment({ attachmentUrl: attachment.url });
```

Comment:
Does this need to be wrapped in a `try` block?
What is the error flow?

Reply:
In the case of an error, an error will be thrown to the error handler, but in the UI side we don't really do anything or display anything since our error scenarios we are just expected to show a broken img